examples/style_sheets/plot_bmh.py

```diff
@@ -1,21 +1,31 @@
 """
-This example demonstrates the "bmh" style, which is the design used in the
-Bayesian Methods for Hackers online book.
+=====================
+Bayesian Methods for Hackers style sheet
+=====================
+
```

Comment:
@NelleV  does the length of the `=`'s need to match the title?

E.g.,

```
=====================
 Bayesian Methods for Hackers style sheet
=====================
```

vs

```
=========================================
 Bayesian Methods for Hackers style sheet
=========================================
```